sdk/examples/wallet/01_create_wallet.rs

```diff
@@ -24,7 +24,7 @@ async fn main() -> Result<()> {
     // Setup Stronghold secret_manager
     let mut secret_manager = StrongholdSecretManager::builder()
         .password(&std::env::var("STRONGHOLD_PASSWORD").unwrap())
-        .build(PathBuf::from("wallet.stronghold"))?;
+        .build(Path::new("wallet.stronghold"))?;
```

Comment:
`build` takes a `AsRef<Path>` anyway so you don't even need a `Path`, just a `&str` should work

Reply:
Can you do a search on the whole repo if we do that elsewhere pls

Reply:
Updated